### PR TITLE
Spelling tests

### PR DESCRIPTION
**Pull request by jsoref**
##### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

##### Component Name

tests

##### Test Plan

The misspellings have been reported at https://github.com/jsoref/netdata/commit/33c5b20ed9511dac0a23ee916fa247cc2f388304#commitcomment-45444734

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/netdata/commit/e2737c090e4b1d7b1bce405b9f702bcf526852a3

##### Additional Information

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

This is split from #10521 per https://github.com/netdata/netdata/pull/10521#issuecomment-813622899

### CODE REVIEW

**Review by vlvkobal — DISMISSED** [review]

File: .github/workflows/review.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           fi
       - name: Run eslint
         if: env.run_eslint == 1
-        uses: reviewdog/action-eslint@v1
+        uses: reviewdog/action-eslint@v1.12.1
```
Comment (ilyam8) [inline .github/workflows/review.yml:28]: perhaps do not include it in the spelling fixes PR?

Comment (jsoref) [reply .github/workflows/review.yml:28]: (Yes, it needs to be dropped, I was trying to convince eslint to work and was throwing things at a wall -- this one failed, but it was time to slepe)

Comment (ilyam8) [reply .github/workflows/review.yml:28]: yeah, it is frustrating that es-lint action hasn't been working for a while and still there is no fix

Comment (jsoref) [reply .github/workflows/review.yml]: I've run into it in a couple of projects, and I have half a 🐶 reviewdog hat of my own, so, I might try to get it (reviewdog/action-eslint#56) fixed.

Obviously w/ my spelling hat on, I don't want to be forced to wait for someone to fix it / fix it myself.

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**